components/common/community/threads/thread/thread.stories.tsx

```diff
@@ -1,43 +1,137 @@
-import * as React from 'react'
-import { Thread } from './thread'
+import React from 'react'
+import { Thread, ThreadProps } from './thread'
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 import { mockData } from '../../../../util/thread_data'
+import useSWR, { mutate } from 'swr'
+import gqlFetcher from '../../../../../utils/gql_fetcher'
+import { gql } from 'graphql-request'
 
 export default {
 	title: 'Common/Community/Thread/Thread',
 	component: Thread,
 } as ComponentMeta<typeof Thread>
 
-const Template: ComponentStory<typeof Thread> = (args) => <Thread {...args} />
+const idByStoryName = {
+	'Thread without title': '63e4055f25783eaaa4f0879c',
+
+	Default: '63e3f79631115da3472a72dc',
+}
+
+const getIdByStoryName = (storyName: string) => {
+	const id = idByStoryName[storyName]
+	return id || idByStoryName.Default
+}
+
+const useUpvoteThread = (moduleID, userProfileID) => {
+	mutate(async () => {
+		fetch('http://emse.dev.joeldesante.com:4000/graphql', {
```

Comment:
Instead of using a hard coded URl here, utilize the API URL that is present in the environment variables